src/crunker.ts

```diff
@@ -204,9 +204,9 @@ export default class Crunker {
    * @param buffer Buffer to export
    * @param type MIME type (default: `audio/wav`)
    */
-  export(buffer: AudioBuffer, type: string = 'audio/wav'): ExportedCrunkerAudio {
+  export(buffer: AudioBuffer, type: string = 'audio/wav', isFloat32: boolean = false): ExportedCrunkerAudio {
```

Comment:
`question`: What do you think about a more user friendly name than `isFloat32`. Could the parameteter be more indicative of it's actual intent?

Reply:
Also, would an enum be more relevant as well with a set of options which may be extended beyond just float32 or not. E.g float16, float32... etc.

Don't know the answer, just an open question.